### DISCUSSION TITLE
In build time I have this error INVALID_DEFAULT_EXPORT

### DISCUSSION THREAD

**Discussion by dmend3z** [vercel/next.js#33308] [Help]
When I build my nextjs project, it starts the next lint, all normal but after saying it was successful it gives this error:

`info  - Compiled successfully
(node:25720) UnhandledPromiseRejectionWarning: Error: INVALID_DEFAULT_EXPORT
    at /home/daniel/WebstormProjects/broker/node_modules/next/dist/build/utils.js:595:23
    at async Span.traceAsyncFn (/home/daniel/WebstormProjects/broker/node_modules/next/dist/trace/trace.js:75:20)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:25720) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 1)
(node:25720) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.

> Build error occurred
Error: INVALID_DEFAULT_EXPORT
    at /home/daniel/WebstormProjects/broker/node_modules/next/dist/build/utils.js:595:23
    at async Span.traceAsyncFn (/home/daniel/WebstormProjects/broker/node_modules/next/dist/trace/trace.js:75:20) {
  type: 'Error'
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
`

I don't know which part of my code may be wrong, if you can help me I would appreciate it.

**Comment by fzhao99** [comment]
Ran into the same error during a build step and realized that I had a file in my `_pages` directory that wasn't exporting anything (hence the INVALID_DEFAULT_EXPORT). In my case, the file wasn't needed anymore and so deleting it fixed the issue.

**Reply by scolobey** [reply]
Thank you.
